ir/node.cpp

```diff
@@ -94,4 +102,24 @@ Util::JsonObject* IR::Node::sourceInfoJsonObj() const {
     return json;
 }
 
+void IR::Node::sourceInfoToJSON(JSONGenerator &json) const {
+    Util::SourceInfo si = srcInfo;
+    unsigned lineNumber, columnNumber;
+    cstring fName = prepareSourceInfoForJSON(si, &lineNumber, &columnNumber);
```

Comment:
Alternately if what makes sense for the 'location' of an assignment is the whole thing rather than just the `=` in the assignment, we should change what the parser puts in the srcInfo for an assignment statement.  Lots of different ways to skin this cat.